### DISCUSSION TITLE
Ignore some models in creating a table during migrate

### DISCUSSION THREAD

**Discussion by jeepers3327** [prisma/prisma#8415] [Q&A] [answered]
Hi,

I have a migration.sql file which is a SQL View which I manually edited. I also create its corresponding model in `schema.prisma`.

```sql
CREATE OR REPLACE VIEW "gist_search" AS
  SELECT g1."id", 
        g1."description",
 		u1."username",
        jsonb_agg(to_jsonb(g2.*)) AS "files", 
        (SELECT count(*) FROM "comments" WHERE "gistId" = g1."id") AS "comment_counts" 
  FROM "gists" g1 INNER JOIN "gist_files" g2 ON g1."id" = g2."gistId" 
 		INNER JOIN users u1 ON u1."id" = g1."authorId"
 GROUP BY g1."id", g2."id", u1."username"
```

```prisma
model GistSearch {
  id            String   @unique
  description   String
  avatarUrl     String
  username      String
  files         Json[]
  commentCounts Int      @map("comment_counts")
  createdAt     DateTime

  @@map("gist_search")
}
```

Now when I try to run `npx prisma migrate dev`, prisma attempts to create a table since the model is present in `schema.prisma`.

Do we have any way to completely ignore the creation of table for this kind of scenario?

**Comment by ryands17** [comment, marked as answer]
@jeepers3327 👋 

Could you check if [`@@ignore`](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#ignore-1) works for your use case?

**Reply by ryands17** [reply]
@jeepers3327 👋 

Did the above work?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
